db/migrate/20200614183600_add_achive_status_to_locations.rb

```diff
@@ -0,0 +1,6 @@
+class AddAchiveStatusToLocations < ActiveRecord::Migration[5.1]
+  def change
+    add_column :locations, :archived, :boolean, default: false, null: false
```

Comment:
You don't need to add an archived boolean because you can just check to see if an archived_at time exists and turn it to nil when it's "unarchived"

Reply:
Completed!